Kills most TGui interfaces

comma opened the pull request:
Kills tgui storage interface
Kills tgui resleever interface
Cleaned up resleever code just a bit cause I couldn't stand looking at it.
Kills tgui jukebox interface
Kills tgui suit jammer interface
Kills tgui brig timer interface

Interfaces ported to NanoUI, storage one removed as unused

<!-- 
Do not forget to add a changelog when you have made admin/player facing changes that can alter gameplay.
Examples which require a changelog entry include:
* Adding/removing objects that players may interact with, or the way they function.
* Adding/removing/altering admin tools.
* Changing the map.

Examples were changelog entries are optional/not typically required:
* Cosmetic changes such as descriptions, sound effects, etc.
* Optimizations and other changes to underlying systems which do not affect gameplay.
* Minor bug fixes.

You'll find a README and example file in .\html\changelogs\ for further instructions.

You can also find a template for adding your changelog directly to the PR description here: https://github.com/Baystation12/Baystation12/wiki/Automatic-changelog-generation
-->

PsiOmegaDelta commented on code/game/machinery/doors/brigdoors.dm:
```diff
@@ -174,39 +174,44 @@
-	if(!src.allowed(usr))
-		return TRUE
+/obj/machinery/door_timer/OnTopic(var/mob/user, var/list/href_list, state)
+	if(..())
```
`OnTopic()` doesn't have to check `..()` unless you're inheriting behavior from a parent.
And if you do, it should've been `if((. = ..()))`

PsiOmegaDelta commented on code/game/machinery/doors/brigdoors.dm:
```diff
@@ -174,39 +174,44 @@
-/obj/machinery/door_timer/ui_act(action, params)
-	if(..())
-		return TRUE
+/obj/machinery/media/jukebox/CanUseTopic(user, state)
```
I assume this should've been `/obj/machinery/door_timer` rather than `/obj/machinery/media/jukebox` ?

PsiOmegaDelta commented on code/game/machinery/doors/brigdoors.dm:
```diff
@@ -174,39 +174,44 @@
+	if (href_list["adjust"])
+		timetoset += text2num(href_list["adjust"])
+		timetoset = Clamp(timetoset, 0, 36000)
+		. = TOPIC_HANDLED
```
Should be `TOPIC_REFRESH` as this particular href may result in an UI change.

comma replied:
It's an autorefreshing interface though, thought that'd be fast enough.

PsiOmegaDelta replied:
From my own experience, the NanoUI interface feels sluggish and annoying to deal with if it doesn't immediately update on interaction.

PsiOmegaDelta commented on code/game/machinery/resleever.dm:
```diff
@@ -99,47 +79,49 @@ obj/machinery/resleever/Process()
-			eject_lace()
-	update_icon()
-	return TRUE
+		return TOPIC_HANDLED
```
As above, should return the value of `..()` if you're calling it

PsiOmegaDelta commented on code/game/objects/items/devices/suit_sensor_jammer.dm:
```diff
@@ -141,29 +135,36 @@ obj/item/device/suit_sensor_jammer/ui_data()
+		var/method = locate(href_list["select_method"]) in suit_sensor_jammer_methods
+		if(method)
+			set_method(method)
+			return TOPIC_HANDLED
```
As before, href's that change values used by the UI should use `TOPIC_REFRESH`

PsiOmegaDelta commented on code/game/machinery/jukebox.dm:
```diff
@@ -118,33 +112,40 @@ datum/track/proc/GetTrack()
-			else if(!current_track)
-				to_chat(usr, "No track selected.")
-			else
+		return TOPIC_HANDLED
```
Again, if you know that a parent type doesn't implement `OnTopic()` then you don't need to call `..()`.
If you do call `..()` then you should return that return value.

comma replied:
Killed all ..() in those ontopics

Roland410 commented on code/modules/mob/language/human/misc/gutter.dm:
```diff
@@ -4,6 +4,6 @@
-	syllables = list ("gra","ba","ba","breh","bra","rah","dur","ra","ro","gro","go","ber","bar","geh","heh", "gra")
-	partial_understanding = list(LANGUAGE_GALCOM = 5, LANGUAGE_HUMAN_RUSSIAN = 20, LANGUAGE_HUMAN_EURO = 20, LANGUAGE_HUMAN_ARABIC = 20, LANGUAGE_HUMAN_IBERIAN = 20, LANGUAGE_HUMAN_CHINESE = 20, LANGUAGE_HUMAN_INDIAN = 20)
+	syllables = list ("yayo", "aiya", "chiksa", "tikat", "bazar","oi", "yo","meit", "grok", "ken", "berk", "cohon", "pendeo")
+	partial_understanding = list(LANGUAGE_GALCOM = 5, LANGUAGE_HUMAN_RUSSIAN = 30, LANGUAGE_HUMAN_EURO = 10, LANGUAGE_HUMAN_ARABIC = 10, LANGUAGE_HUMAN_IBERIAN = 30, LANGUAGE_HUMAN_CHINESE = 20, LANGUAGE_HUMAN_INDIAN = 10)
```
Did you commit this on purpose.. or?

comma replied:
Nope, removed

comma committed:
Kills most tgui

Kills tgui storage interface
Was completely unused.
Kills tgui resleever interface

Ports to NanoUI
Cleaned up resleever code just a bit cause I couldn't stand looking at it.
Kills tgui jukebox interface

Ported to NanoUI
Kills tgui suit jammer interface

Ported to NanoUI
Kills tgui brig timer interface

Ported to NanoUi
